src/main/java/uk/gov/pay/ledger/performance/dao/PerformanceDao.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.pay.ledger.performance.dao;
+
+import org.jdbi.v3.core.Jdbi;
+import uk.gov.pay.ledger.performance.dao.mapper.PerformanceReportEntityMapper;
+import uk.gov.pay.ledger.performance.entity.PerformanceReportEntity;
+
+import javax.inject.Inject;
+
+public class PerformanceDao {
+
+    private static final String PERFORMANCE_REPORT =
+            "select coalesce(count(amount),0) as volume, coalesce(sum(amount),0) total_amount, " +
+                    "coalesce(avg(amount),0) avg_amount from transaction " +
+            "where state='SUCCESS' and type='PAYMENT' AND  transaction_details @> '{\"live\": true}'";
```

Comment:
This should use the `live` column on the transaction directly -- having a condition on a non-indexed attribute would end up costing a lot.

The `live` column is populated from this value when the transaction is made (and has a partial index https://github.com/alphagov/pay-ledger/pull/322)

Reply:
Oh! I missed the "live" column.